site/src/types/types.ts

```diff
@@ -28,7 +28,7 @@ export interface CourseData {
 
 export interface ProfessorData {
     name: string;
-    shortened_name: string;
+    shortenedName: string;
```

Comment:
Both CourseData and ProfessorData seem to be unused so feel free to remove them.